housing.html

```diff
@@ -126,27 +126,25 @@ <h1>HOUSING TOURS</h1>
 
                     <div class="col-sm-5 col-sm-offset-0">
                         <div class="section-title">
-                            <h1 class="title">Unit 2</h1>
+                            <h1 class="title">Unit 3</h1>
```

Comment:
We should have both tours (Unit 2 & Unit 3). You can look at the carousel on the home page for an example of how to add multiple videos. I think youll also need to add the glyphicons (the l/r arrows). You can update the language on the sidebar to refer to general units.